SDK Showcase/SDK Showcase/Interactors/ResourceInteractor.swift

```diff
@@ -4,44 +4,75 @@ import OneginiSDKiOS
 import SwiftUI
 
 protocol ResourceInteractor {
-    func fetchDeviceList()
-    func fetchImplicit()
+    /// Requests performed on UserClient
+    func sendAuthenticatedRequest()
+    func sendImplicitRequest()
+    
+    /// Requests performed on UserDevice
+    func sendUnauthenticatedRequest()
+    func sendAnonymousRequest()
 }
 
 class ResourceInteractorReal: ResourceInteractor {
     private let userClient = SharedUserClient.instance
+    private let deviceClient = SharedDeviceClient.instance
     private let decoder = JSONDecoder()
     @ObservedObject var app: ShowcaseApp
 
     init(app: ShowcaseApp) {
         self.app = app
     }
     
-    func fetchDeviceList() {
+    func sendUnauthenticatedRequest() {
+        // Should be defined on the access
+        let pathToTheResource = "path-to-the-resource"
+        let request = ResourceRequestFactory.makeResourceRequest(path: pathToTheResource)
+        deviceClient.sendUnauthenticatedRequest(request) { [weak self] response, error in
+            if let error = error {
+                self?.handleError(error)
+            } else {
+                self?.handleData("unauthenticated request for the resource has been fetched.")
+            }
+        }
+    }
+    
+    func sendAnonymousRequest() {
+        let pathToTheResource = "application-details"
```

Comment:
It is used in one place, why not to inline?

Reply:
does not matter